metagraph/src/common/elias_fano.cpp

```diff
@@ -14,29 +14,29 @@
 namespace mtg {
 namespace common {
 
-void concat(const std::vector<std::string> &files, const std::string &result) {
+std::vector<std::string> concat(const std::vector<std::string> &files, const std::string &result) {
     if (files.empty())
-        return;
+        return {};
+    std::vector<std::string> original_files;
```

Comment:
Please add an empty line after returns
```suggestion
        return {};

    std::vector<std::string> original_files;
```